pkg/controller/k8srole.go

```diff
@@ -9,21 +9,22 @@ import (
 
 type k8srole struct {
 	*DependentResourceHelper
+	reconciler *BaseGenericReconciler
 }
 
 func (res k8srole) Update(toUpdate runtime.Object) (bool, error) {
 	return false, nil
 }
 
 func (res k8srole) NewInstanceWith(owner v1alpha2.Resource) DependentResource {
-	return newOwnedk8srole(owner)
+	return newOwnedk8sRole(res.reconciler, owner)
```

Comment:
`newOwnedk8sRole` => `newOwnedK8sRole` would be better, no?

Reply:
Same for other function names, actually…

Reply:
I agree with you ;-)